lib/gds_api/performance_platform/data_out.rb

```diff
@@ -43,6 +43,119 @@ class DataOut < GdsApi::Base
       def service_feedback(transaction_page_slug)
         get_json("#{endpoint}/data/#{transaction_page_slug}/customer-satisfaction")
       end
+
+      # Fetches data from the performance platform for a given page slug
```

Comment:
Ah, you've changed the implementation to one method since I reviewed.  Might be worth listing some examples of the transactions, group_by, and collect params to help folk understand what kinds of calls they can make.  (Maybe we can link to an API doc for backdrop read if such a thing exists?).